### ISSUE TITLE
Issue displaying Matches

### ISSUE THREAD

**Issue by c0mat0se** [myserywork/pubg-api-php#2]
I am having some issues fetching matches. I have setup the example just like in your apitest.php.

Unfortunately, it always results in Page_NOT_FOUND for matches. I can get status to work, and if I plug in a game id, I can fetch that, but for matches I cannot. I have tested this with multiple IDs, regions, and so on. My code I am using for the variables is posted below:

`$plataform = 'pc-na';
$sort = '-createdAt';
$limit = 5; 
$offset = 0; 
$filter_start = date(DateTime::ISO8601);
$filter_end = date(DateTime::ISO8601);
$player_ids = array('76561198027798651'); 
$game_mode = 'squad'; // solo,squad ... 
print_r($pubg->matches($plataform,$sort,$limit,$offset,$filter_start,$filter_end,$player_ids,$game_mode));`

Any help would be appreciated!

**Comment by cainiaoDJ** [comment]
use your id (76561198027798651) in pubg's doc, it returns 
```
{
  "errors": [
    {
      "title": "Not Found",
      "detail": "No players found matching criteria"
    }
  ]
}
```

So I think https://documentation.playbattlegrounds.com doesn't work like his game. 
Boring

**Comment by c0mat0se** [comment]
Yeah interestingly enough when I convert it over to how the documentation example does it, it works fine.
